testutils/collection/data.go

```diff
@@ -285,6 +285,34 @@ var (
 			AvailabilityZone: "default",
 			ExportLocations:  []string{"192.168.100.101"},
 		},
+		{
```

Comment:
why to add new objects? Can we not reuse the existing?

Reply:
Replied in different PR

Reply:
#1299